tests/view_tests/tests/test_debug.py

```diff
@@ -271,7 +272,8 @@ def test_template_not_found_error(self):
         self.assertContains(response, '<div class="context" id="', status_code=500)
 
 
-class ExceptionReporterTests(SimpleTestCase):
+@override_settings(ROOT_URLCONF='view_tests.urls')
+class ExceptionReporterTests(LoggingCaptureMixin, SimpleTestCase):
```

Comment:
`LoggingCaptureMixin` isn't related, I originally was adding tests to make a 404 and test for the cleansing of those request.META when I realized that 404 doesn't pass back request.META. I forgot to remove and am unsure of why I added it in the first place.